client/public/index.html

```diff
@@ -40,7 +40,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
       href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap"
-      rel="stylesheet"
+      rel="preload" as="style" onload="this.onload=null;this.rel='stylesheet'"
```

Comment:
Maybe something like this would suffice:
```suggestion
      rel="preload" as="style"
```